### PR TITLE
Add DeviceMessage property to BeaconEvent

### PR DESCRIPTION
**Pull request by okdtsk**
Support device message from LINE beacon.
spec: https://devdocs.line.me/en/#webhook-event-object

### CODE REVIEW

**Comment by okdtsk** [comment]
As of 2017/4/26, `flake8-docstrings` has dependency issue related `pydocstyle`.
So test `py35-flake8-src` will be failed but #39 will fix this problem.

File: linebot/models/events.py [hunk 1]
```diff
@@ -282,14 +282,16 @@ class Beacon(Base):
...
         :param str type: Type of beacon event
         :param str hwid: Hardware ID of the beacon that was detected
+        :param str dm: Optional. Device message of beacon that was detected in bytearray
         :param kwargs:
         """
         super(Beacon, self).__init__(**kwargs)
 
         self.type = type
         self.hwid = hwid
+        self.dm = bytearray.fromhex(dm) if dm is not None else None
```
Comment (be-hase) [inline linebot/models/events.py]: How about using `device_message` instead of `dm`.
Other SDK adopt `device_message(or deviceMessage)`

Comment (okdtsk) [inline linebot/models/events.py]: Thanks for your review 🌱  I agree with your suggestion, I'll change that property's name.

**Comment by okdtsk** [comment]
I add another test when there is no 'dm' key inside webhook's json.

**Comment by be-hase** [comment]
LGTM 👍

**Comment by okdtsk** [comment]
@be-hase Thanks! I rebased this feature branch onto master (for #39) and squashed into one commit.

**Comment by okdtsk** [comment]
I found a failed test `py35-flake8-other` due to E501 - line too long at `examples/flask-kitchensink/app.py:253:100`, so fixed it.

```diff
diff --git c/examples/flask-kitchensink/app.py w/examples/flask-kitchensink/app.py
index b104be5..f6f2d7f 100644
--- c/examples/flask-kitchensink/app.py
+++ w/examples/flask-kitchensink/app.py
@@ -250,7 +250,8 @@ def handle_postback(event):
 def handle_beacon(event):
     line_bot_api.reply_message(
         event.reply_token,
-        TextSendMessage(text='Got beacon event. hwid={}, device_message={}'.format(event.beacon.hwid, event.beacon.device_message)))
+        TextSendMessage(text='Got beacon event. hwid={}, device_message={}'
+                             .format(event.beacon.hwid, event.beacon.device_message)))


 if __name__ == "__main__":
```

**Pull request merged**